sql/table.sql

```diff
@@ -406,6 +406,7 @@ CREATE TABLE IF NOT EXISTS `phenotype_feature_attrib` (
 @column stable_id            Ensembl stable identifier for the phenotype
 @column name                 Phenotype short name. e.g. "CAD".
 @column description varchar  Phenotype long name. e.g. "Coronary Artery Disease".
+@column class_attrib_id      Phenotype type attribute id. e.g. for attribute non_specified
 
```

Comment:
This could be more specific, less attributey, for example - 'Class of phenotype entry, eg trait, unspecified, tumour - used for filtering'

Reply:
Thanks @sarahhunt, yes, indeed. Have updated it in the next commit.